scripts/install-mender.sh

```diff
@@ -98,15 +117,22 @@ usage() {
     echo "usage: install-mender.sh [options] [component...] [-- [options-for-mender-setup] ]"
     echo ""
     echo "options: [-h, help] [-c channel] [--demo] [--commercial]"
-    echo "  -h, --help          print this help"
-    echo "  -c CHANNEL          channel: stable(default)|experimental"
-    echo "  --demo              use defaults appropriate for demo"
-    echo "  --commercial        install commercial components, requires --jwt-token"
-    echo "  --jwt-token TOKEN   Hosted Mender JWT token"
+    echo "  -h, --help             print this help"
+    echo "  -c CHANNEL             channel: stable(default)|experimental"
+    echo "  --demo                 use defaults appropriate for demo"
+    echo "  --commercial           install commercial components, requires --jwt-token"
+    echo "  --force-mender-client4 install the Mender Client 4.x serie on all the distros"
```

Comment:
Here and below: I think the right grammar is "series". Better to check with an English native :+1: